server/re/include/irods_re_structs.hpp

```diff
@@ -86,4 +86,27 @@ packReiAndArg( ruleExecInfo_t *rei, char *myArgv[],
 int
 unpackReiAndArg( rsComm_t *rsComm, ruleExecInfoAndArg_t **reiAndArg,
                  bytesBuf_t *packedReiAndArgBBuf );
+
+int copyRuleExecInfo( ruleExecInfo_t *from, ruleExecInfo_t *to );
+
+int freeRuleExecInfoStruct( ruleExecInfo_t *rs, int freeSpeialStructFlag );
```

Comment:
freeSpeialStructFlag -> freeSpecialStructFlag ?

here and two lines below as well...